components/uniset/src/error.rs

```diff
@@ -0,0 +1,46 @@
+// This file is part of ICU4X. For terms of use, please see the file
+// called LICENSE at the top level of the ICU4X source tree
+// (online at: https://github.com/unicode-org/icu4x/blob/master/LICENSE ).
+use std::fmt;
+use std::path::{Path, PathBuf};
+
+#[derive(Debug)]
+pub enum Error {
+    Io(std::io::Error, Option<PathBuf>),
+    PpucdParse(PpucdParseError),
+}
+
+#[derive(Debug, PartialEq, Copy, Clone)]
+pub struct PpucdParseError {
```

Comment:
The uniset crate shouldn't know what ppucd is.  Why do you have a ppucd error in the uniset crate?

Reply:
Done. Removed. Even during the old misunderstading of how PPUCD Provder should work, this would have been code that got put in the wrong place and got lost.